src/Microsoft.ServiceFabric.Actors.KVSToRCMigration/Microsoft.ServiceFabric.Actors.KVSToRCMigration.csproj

```diff
@@ -29,9 +29,13 @@
   </ItemGroup>
   <ItemGroup>
     <!-- Overriding versions of transitive dependencies -->
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
-    <PackageReference Include="System.Security.Cryptography.Xml" Version="4.7.1" />
-    <PackageReference Include="System.Text.Encodings.Web" Version="4.5.1" />
+    <!-- 
+      Some versions of transitive dependencies have to be overwritten, since the default versions
+      defined by the direct dependencies are marked as vulnerable packages on nuget.org.
+    -->
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
+    <PackageReference Include="System.Security.Cryptography.Xml" Version="9.0.0-rc.1.24431.7" />
```

Comment:
We should stick to the latest stable versions of the packages. I'm not sure if we can use the 9.0 versions at this point, but since they are still pre-release, it's a moot point. 

Here is what pre-release means: https://semver.org/#spec-item-9

Reply:
@olegsych thank you for this insight. My reasoning for using the prerelease versions was that using the latest stable version of `System.Security.Cryptography.Xml` introduced another transitive vulnerable dependency: `System.Formats.Asn1:8.0.0`, which is not present in the prerelease version of `System.Security.Cryptography.Xml`. 

I reverted all packages to their latest table version, and resolved this vulnerable transitive dependency by, again, overriding it's verion:
```
<PackageReference Include="System.Formats.Asn1" Version="8.0.1" />
```

Nevertheless, it is very strange to me that we are encountering so many of these vulnerable transitive dependencies.